### DISCUSSION TITLE
New G14 2022 6800s owner .. eGPU controls?

### DISCUSSION THREAD

**Discussion by monty77uk** [seerge/g-helper#348] [General]
Hello!

Looking forward to using tool when my G14 arrives.  Considering hooking this up to my 3080 gaming box for docked gaming - anything built in to help with the usual eGPU woes of disabling existing dGPU etc or other settings to optimize?  

Appreciate this is a very niche use case right now with USB4 being beta only.

Thanks!

**Comment by seerge** [comment]
You can disable existing dGPU by setting Eco mode.

**Reply by monty77uk** [reply]
Awesome.  Does it have any impact on CPU power limits?  Obviously in egpu mode I'm looking for max CPU 👍

Thanks!

**Reply by seerge** [reply]
It doesn't

**Reply by monty77uk** [reply]
Perfection, thank you!

**Reply by monty77uk** [reply]
I have the eGPU talking to the G14 now .. it was bumpy though.  It would recognize the eGPU but then disconnect it from device manager and cause the laptop to BSOD or hang.  It seemed to go away when I disabled 'keep gpu disabled on USB-C charger in Optimized mode' .. which would make sense given eGPU is seen as a usb-C charger, however I wasn't in optimized mode - I was using Eco.  Did I misunderstand what that option does?

**Reply by seerge** [reply]
@monty77uk checkbox you refer to affects only "Optimized" mode, when laptop will be plugged to charge over usb-c (not barrel adapter) it won't enable dedicated GPU (as it would do otherwise).

Since you experiment with not related 3rd party hardware it will be easeir to either constantly keep dGPU on (Standard) or off (Eco).

**Reply by monty77uk** [reply]
Totally agree .. I'll keep away from Optimized.  Love G-helper by the way, so much cleaner than AC!!

**Reply by monty77uk** [reply]
I know I'm into the realms of custom stuff here that the tool was never intended to consider but how does it know which GPU to disable when you go into Eco mode?  With multiple GPUs it seems a bit random as to which one it'll disable when you choose Eco mode so I'm sticking with standard.... but some titles complain when there are multiple active GPUs.

**Reply by seerge** [reply]
Asus has a built in method for diabling/enabling own dGPU on hardware level. So it disables only that. Your eGPU has nothing to do with g-helper at all :) and it doesn't touch it

**Reply by monty77uk** [reply]
...assuming Asus does it gracefully - I doubt they thought about eGPUs, nor should they have.  Definitely seems like it's not always disabling the 6800S - it's fine, just for some games I'll need to go into device manager and disable the 6800 manually to avoid issues - no biggy :-)
